app/src/main/java/com/brainwallet/tools/manager/PromptManager.java

```diff
@@ -61,7 +61,7 @@ public boolean shouldPrompt(Context app, PromptItem item) {
             case PAPER_KEY:
                 return !BRSharedPrefs.getPhraseWroteDown(app);
             case UPGRADE_PIN:
-                return BRKeyStore.getPinCode(app).length() != 6;
```

Comment:
This check is everywhere. Why not make it a global constant and test @andhikayuana ? 
Something like `BRKeyStore.getPinCode(app).length() = BW_PIN_LENGTH`

Then we can add a test as well.

Thoughts?

Reply:
thinking about this before, will change it